api_v3/lib/KalturaErrors.php

```diff
@@ -819,5 +819,15 @@ class KalturaErrors extends APIErrors
 	const HOST_NAME_CONTAINS_ASTRIX = "HOST_NAME_CONTAINS_ASTRIX;HOST_NAME;Host name contains *, use # instead {@HOST_NAME@}";
 	const CHANGE_DESCRIPTION_CANNOT_BE_EMPTY = "CHANGE_DESCRIPTION_CANNOT_BE_EMPTY;;Param changeDescription cannot be empty";
 	const SEARCH_ITEM_TYPE_NOT_FOUND = 'SEARCH_ITEM_TYPE_NOT_FOUND;SEARCH_ITEM_TYPE,ELASTIC_FIELD_NAME; Search item type [@SEARCH_ITEM_TYPE@] not found for field: [@ELASTIC_FIELD_NAME@]';
+	
+	/**
```

Comment:
I believe that all the classes that extend the KalturaErrors are plugins
Since this is a core api service, not sure where to place the extended class (unlike plugin which is placed in the correlative plugin folder)